docker-compose.yml

```diff
@@ -47,14 +47,8 @@ services:
       - '8092:8080'
     networks:
       - opentosca
-  engine-ia-jdk8:
```

Comment:
Do we remove the JDK 8 support completely now?

Reply:
hope so 👀